core/2d/ActionCoroutine.h

```diff
@@ -0,0 +1,158 @@
+/****************************************************************************
+Copyright (c) 2008-2010 Ricardo Quesada
+Copyright (c) 2010-2012 cocos2d-x.org
+Copyright (c) 2011      Zynga Inc.
+Copyright (c) 2013-2016 Chukong Technologies Inc.
+Copyright (c) 2017-2018 Xiamen Yaji Software Co., Ltd.
+Copyright (c) 2023      lich426 (lich426@gmail.com)
```

Comment:
```suggestion
Copyright (c) 2019-present Axmol Engine contributors (see AUTHORS.md).
```

Reply:
I will do:

1. Rename `AUTHORS` -- `AUTHORS-OLD`
2. Add `AUTHORS.md` for new authors list(of axmol) and `lich426 (lich426@gmail.com)` will be in the list